examples/inference-deployments/mpt/mpt_7b_handler.py

```diff
@@ -63,10 +63,33 @@ def _parse_inputs(self, inputs: Dict[str, Any]):
 
         return generate_input, generate_kwargs
 
-    def predict(self, **inputs: Dict[str, Any]):
-        generate_input, generate_kwargs = self._parse_inputs(inputs)
-        outputs = self.generator(generate_input, **generate_kwargs)
-        return [output[0]['generated_text'] for output in outputs]
+    def _extract_output(self, outputs: List[Any]):
+        output_list = []
+        for output in outputs:
+            output_bytes = output[0]['generated_text']
+            output_list.append(output_bytes)
+        return output_list
+
+    def predict(self, input_dicts: List[Dict[str, Any]]):
```

Comment:
we should have this match the hf_model handler (https://github.com/mosaicml/mcloud/blob/dev/inference_go/model_server/hf_model.py#L146-L168)

Reply:
done